doc/build.md

```diff
@@ -135,9 +159,9 @@ Examples:
 
 It is up to the user to select the CMake Generator of choice, but we encourage the use of the Ninja generator. To use the Ninja Generator, you need to first install the Ninja binary from https://ninja-build.org/
 
-You then need to set the generator to ```Ninja``` and the ```CMAKE_MAKE_PROGRAM``` variable to the Ninja binary you downloaded.
+You then need to set the generator to ```Ninja```.
 ```
-python build.py --generator Ninja --build-args=-DCMAKE_MAKE_PROGRAM='path to ninja binary'
+python build.py --generator Ninja
```

Comment:
Specifying the CMAKE_MAKE_PROGRAM should not be necessary as CMake will figure out which one to use based on the generator. This option mostly seems to be used to force a specific "make" program.